code/API_definitions/qod-api.yaml

```diff
@@ -290,6 +290,69 @@ paths:
         - threeLegged:
             - "qod-sessions-delete"
 
+  /sessions/{sessionId}/extend:
+    post:
+      tags:
+        - Change session duration
+      summary: "Extend the duration of an active session"
+      description: |
+        Extend the duration of an active QoS session. If this operation is executed successfully, the new duration of the target session will be the original duration plus the additional duration. The remaining duration plus the requested additional duration shall not exceed the maximum limit. Otherwise, the additional duration will be the maximum limit minus the remaining duration.
+      operationId: extendQosSessionDuration
+      parameters:
+        - name: sessionId
+          in: path
+          description: Session ID that was obtained from the createSession operation
+          required: true
+          schema:
+            $ref: "#/components/schemas/SessionId"
+      requestBody:
+        description: Parameters to extend the duration of an active session
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/ExtendSessionDuration"
+        required: true
+      responses:
+        "200":
+          description: Contains information about active session
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SessionInfo"
```

Comment:
Returning full session info here seems excessive.

Why not return only the fields that have changed (`duration`, `expiresAt`) plus `sessionId` itself?

Reply:
I would see acceptable to return the new modified SessionInfo, no strong opinion on this

Reply:
Agreed within meeting on Nov 16th to return full SessionInfo.